README.md

```diff
@@ -35,11 +35,11 @@ SPDX-License-Identifier: Apache-2.0
         ```
     - To run, taking black_scholes for example:
         ```bash
-        $  python -c "import dpbench; dpbench.run_benchmark(\"black_scholes\")" 2> /dev/null
+        $  dpbench -b black_scholes run
```

Comment:
We should remove the leading `$` sign. It gets copied along with the command when we copy it to clipboard.

Reply:
It is nice to fix but unrelated to this PR. Let's keep it consistent till it is resolved in separate PR.